db/migrate/20170118232430_create_join_table_authentication_configuration_script_base.rb

```diff
@@ -0,0 +1,9 @@
+class CreateJoinTableAuthenticationConfigurationScriptBase < ActiveRecord::Migration[5.0]
+  def change
+    create_table :authentication_configuration_script_bases do |t|
+      t.bigint  :authentication_id
+      t.bigint  :configuration_script_id
+      t.index   :configuration_script_id, :name => 'configuration_script_id'
```

Comment:
Do we care cost comes with the index? If we do, my take on this is we are more likely to look up a JobTemplate's credentials than the other way around.

We need to specify the index name this time because the auto-generated one is blowing up the name limit.